client/webserver/site/src/js/registry.ts

```diff
@@ -376,6 +376,7 @@ export interface PageElement extends HTMLElement {
   checked?: boolean
   href?: string
   htmlFor?: string
+  animating?: boolean
```

Comment:
Please don't add custom attributes to elements.

Reply:
Changed as was suggested in https://github.com/decred/dcrdex/pull/1921#discussion_r1017804446.